### ISSUE TITLE
Friendly note about missing accessibility

### ISSUE THREAD

**Issue by TTomczek** [TTomczek/studygroupmap#48]
Hi there! I really appreciate the effort put into this project. I wanted to kindly point out that there seems to be a lack of accessibility features. Enhancing accessibility would make the project even better and more inclusive for everyone. Thanks for considering this!
